### DISCUSSION TITLE
A bookmark card concept

### DISCUSSION THREAD

**Discussion by sandsoncosta** [hbstack/.github#86] [Ideas]
I saw a bookmark card on a blog and I thought the idea was really interesting and I thought it would be a good idea to have a tag to enrich the theme.

The blog: https://skiptotheendpoint.co.uk/dot-slash-administrator-a-security-risk-analysis/

![image](https://github.com/user-attachments/assets/621d5adc-c6e8-42e1-a0d2-412d92b89271)

I did a custom on my blog and got a very interesting end result.

![image](https://github.com/user-attachments/assets/f37d8e61-080a-4274-b109-3528a88c00e6)

![image](https://github.com/user-attachments/assets/4e2e0d13-3d0b-4f67-9760-e0b69faa8a38)

I thought it looked really interesting. Now in my case, I changed the color manually. I actually copied and pasted the html and css.

![image](https://github.com/user-attachments/assets/417ee6c4-85a0-4ab0-a12e-7d6eae3bdf12)

But the idea is that it would be something like this:

![image](https://github.com/user-attachments/assets/fb82c68e-351b-48a7-a454-d1a203177948)

And it would automatically stay on that card. If the theme is black, the white card, if it is light, the black card. Or dark and dark and light and light.

![image](https://github.com/user-attachments/assets/680b7530-5e16-4118-abfb-882290f5c5eb)

![image](https://github.com/user-attachments/assets/46fd901b-810c-414c-bae8-117f84052279)

Set a size, choose whether to center or not.

![image](https://github.com/user-attachments/assets/964c8812-58c5-4464-bba3-89cd9d8e96a4)

**Comment by razonyang** [comment]
It looks useful, please track this feature on hugomods/bootstrap#163.

**Comment by razonyang** [comment]
Available since [github.com/hugomods/bootstrap@v0.23.0](https://github.com/hugomods/bootstrap/releases/tag/v0.23.0), please check the [PR](https://github.com/hugomods/bootstrap/pull/164) for how to use it.

> Set a size, choose whether to center or not.

The request above doesn't make sense to me, it always to take full width of parent element, if you want to make it to take less spaces, please wrap it with Bootstrap grid, for example.

```markdown
{{< bs/col offset="0 lg:2" size="12 lg:8" >}}
{{< bs/bookmark-card ... >}}
{{< /bs/col >}}
```

Tweak the size and offset to meet your needs.

**Reply by sandsoncosta** [reply]
Very good! Perfect!
![image](https://github.com/user-attachments/assets/6ae64ae4-33ca-414e-8df1-3115715f15a9)

Thanks for feature!

**Reply by razonyang** [reply]
No problem, BTW, you can apply utilities on the card, for example, to add gap between card and next paragraph.

```markdown
{{< bs/col class="mb-2" offset="0 lg:2" size="12 lg:8" >}}
...
...
```
